strings.txt

```diff
@@ -9013,35 +9013,36 @@
   [cuisine_coffee_shop]
     tags = android, ios
     comment = Cuisine
-    en = coffee shop
-    ru = кафе
-    uk = кав'ярня
-    tr = kafe
-    es = cafetería
-    sv = kafé
-    sk = kaviareň
-    pt = cafeteria
-    ro = cafenea
-    pl = kawiarnia
-    ko = 커피 샵
-    id = kafe
-    hu = kávézó
-    de = Café
-    fr = café
-    nl = koffiehuis
-    fi = kahvila
-    da = café
-    cs = kavárna
-    it = bar caffetteria
-    nb = kafé
-    zh-Hant = 咖啡_館
-    zh-Hans = 咖啡_馆
-    vi = tiệm_cà phê
-    th = ร้านกาแฟ
-    ja = コーヒーショップ
-    el = καφετέρια
-    ar = مقهى
-    he = בית קפה
+    en = Coffee
```

Comment:
Да, именно так. А пока они берутся на айфоне и десктопе из ядра, а на андроиде из стрингов.

> On 18.3.2016, at 16.29, Ilya Zverev <notifications@github.com> wrote:
> 
> In strings.txt:
> 
> > -    fr = café
> > -    nl = koffiehuis
> > -    fi = kahvila
> > -    da = café
> > -    cs = kavárna
> > -    it = bar caffetteria
> > -    nb = kafé
> > -    zh-Hant = 咖啡_館
> > -    zh-Hans = 咖啡_馆
> > -    vi = tiệm_cà phê
> > -    th = ร้านกาแฟ
> > -    ja = コーヒーショップ
> > -    el = καφετέρια
> > -    ar = مقهى
> > -    he = בית קפה
> > +    en = Coffee
> «В стрингах переводы остались виеменно, только для андроида. Они должны боаться из json файлов через ядро. Дима пока не добрался до этого изменения» — то есть, после того, как Дима доберётся, нужно будет выпилить переводы cuisine из стрингов и ресурсов для айфона и андроида?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>